scripts/us_eia/eia_860/test_data/2_plant.csv

```diff
@@ -1,19 +1,19 @@
+"eia/pp/663","\"Deerhaven Generating Station\"","\"10001 NW 13th Street, Gainesville, FL 32653\"","dcid:geoId/12","dcid:zip/32653","29.7592","-82.3878","NAICS/22","663","dcid:eia/u/6909","dcid:NonRegulated","dcid:EIA_ElectricUtility","dcid:FERC_NonQualifiedCogenerator","\"QF12-380-000;QF03-140-000\"","dcid:FERC_NonQualifiedSmallPowerProducer","","dcid:FERC_NonQualifiedExemptWholesaleGenerator","","","","dcid:EIA_Operating","[138.0 KiloVolt]","","","dcid:False","dcid:False","dcid:False"
+"eia/pp/664","\"John R Kelly\"","\"515 SE 5 Ave, Gainesville, FL 32601\"","dcid:geoId/12","dcid:zip/32601","29.6461","-82.3208","NAICS/22","664","dcid:eia/u/6909","dcid:NonRegulated","dcid:EIA_ElectricUtility","dcid:FERC_NonQualifiedCogenerator","","dcid:FERC_NonQualifiedSmallPowerProducer","","dcid:FERC_NonQualifiedExemptWholesaleGenerator","","","","","[138.0 KiloVolt]","[12.47 KiloVolt]","","dcid:False","dcid:False","dcid:False"
 "eia/pp/665","\"G W Ivey\"","\"675 N. Flagler Avenue, Homestead, FL 33030\"","dcid:geoId/12","dcid:zip/33030","25.4756","-80.47","NAICS/22","665","dcid:eia/u/8795","dcid:NonRegulated","dcid:EIA_ElectricUtility","dcid:FERC_NonQualifiedCogenerator","","dcid:FERC_NonQualifiedSmallPowerProducer","","dcid:FERC_NonQualifiedExemptWholesaleGenerator","","","","","[13.2 KiloVolt]","","","dcid:False","dcid:False","dcid:False"
-"eia/pp/666","\"J D Kennedy\"","\"4215 Talleyrand Ave., Jacksonville, FL 32206\"","dcid:geoId/12","dcid:zip/32206","30.364818","-81.625967","NAICS/22","666","dcid:eia/u/9617","dcid:NonRegulated","dcid:EIA_ElectricUtility","dcid:FERC_NonQualifiedCogenerator","","dcid:FERC_NonQualifiedSmallPowerProducer","","dcid:FERC_NonQualifiedExemptWholesaleGenerator","","","","","[69 KiloVolt]","[138 KiloVolt]","[26 KiloVolt]","dcid:False","dcid:False","dcid:False"
-"eia/pp/63917","\"DC Water Solar\"","\"5000 Overlook Ave. SW, Washington, DC 20032\"","dcid:geoId/11","dcid:zip/20032","38.819244","-77.018944","NAICS/22","63917","dcid:eia/u/63572","dcid:Regulated","dcid:EIA_IndependentPowerProducer_NonCombinedHeatPower","dcid:FERC_NonQualifiedCogenerator","","dcid:FERC_QualifiedSmallPowerProducer","\"QF20-157-000\"","dcid:FERC_NonQualifiedExemptWholesaleGenerator","","","","","[69 KiloVolt]","","","dcid:False","dcid:False","dcid:False"
+"eia/pp/666","\"J D Kennedy\"","\"4215 Talleyrand Ave., Jacksonville, FL 32206\"","dcid:geoId/12","dcid:zip/32206","30.364818","-81.625967","NAICS/22","666","dcid:eia/u/9617","dcid:NonRegulated","dcid:EIA_ElectricUtility","dcid:FERC_NonQualifiedCogenerator","","dcid:FERC_NonQualifiedSmallPowerProducer","","dcid:FERC_NonQualifiedExemptWholesaleGenerator","","","","","[69.0 KiloVolt]","[138.0 KiloVolt]","[26.0 KiloVolt]","dcid:False","dcid:False","dcid:False"
```

Comment:
hmm... this might be a good test for Quantity parsing, whether the resulting node with and without .0 is the same...